test/runtests.jl

```diff
@@ -19,8 +18,16 @@ using Test
     train_rows, test_rows = partition(eachindex(y), 0.7, shuffle=true)
     fit!(regressor; rows=train_rows, verbosity=0)
 
-    @test ==(
-        mean(predict(regressor; rows=test_rows)),
-        predict_mean(regressor; rows=test_rows),
+    @test predict_joint(regressor; rows=test_rows) isa AbstractGPs.AbstractMvNormal
+    @test predict(regressor; rows=test_rows) isa Vector{<:AbstractGPs.Normal}
+    @test predict_mean(regressor; rows=test_rows) isa Vector{<:Real}
+
+    @test isapprox(
+        mean(only(predict(regressor; rows=test_rows[2:2]))),
+        mean(predict(regressor; rows=test_rows)[2]),
+    )
```

Comment:
@willtebbutt Sorry, this test does not convince me. What we need is something like:
 ```
  @test isapprox(
         mean(predict(regressor; rows=test_rows[1:10])[j])),
         mean(predict(regressor; rows=test_rows)[j]),
     )
```
for any `j in 1:10`. This is what I call "consistency" in [this comment](https://github.com/alan-turing-institute/MLJ.jl/issues/552#issuecomment-673744549) .

Reply:
No problem. Test updated.